### ISSUE TITLE
`no-array-callback-reference` false positive when using ternary

### ISSUE THREAD

**Issue by William5553** [sindresorhus/eslint-plugin-unicorn#2288]
`no-array-callback-reference` has a false positive when using a ternary operator for filter()

```js
return matchData.cycles.filter(
	isTeleop() 
		? cycle => cycle.score.time >= 18 
		: cycle => cycle.score.time < 18
).length
```
